src/include/units/concepts.h

```diff
@@ -80,14 +82,11 @@ struct scaled_unit;
 // TODO: Remove when P1985 accepted
 namespace detail {
 
-struct is_derived_from_scaled_unit_impl {
-  template<ratio R, typename U>
-  static constexpr std::true_type check_base(const scaled_unit<R, U>&);
-  static constexpr std::false_type check_base(...);
-};
+template<ratio R, typename U>
+void to_base_scaled_unit(const volatile scaled_unit<R, U>*);
 
 template<typename T>
-inline constexpr bool is_derived_from_scaled_unit = decltype(is_derived_from_scaled_unit_impl::check_base(std::declval<T>()))::value;
+concept is_derived_from_scaled_unit = requires { to_base_scaled_unit(std::declval<const volatile T*>()); };
```

Comment:
Please incorporate those sub-concepts into the main ones (i.e. `Unit`). The prefix `is_` in the C++ standard library is reserved for traits and it might be misleading what you work with.

Reply:
Considering my other comment, do you still think I should drop the `is_` prefix? They being `concept`s is an implementation detail to make them work on MSVC.